### PR TITLE
Updating cyclegan identity loss

### PR DESCRIPTION
**Pull request by bearpelican**
I think the identity losses were flipped.

The original code has id_loss = ||G(A) - A||
https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/690470c80bb7021d96c9d3f82e416e633dbb2da3/models/cycle_gan_model.py#L158

The notebook had ||G(A) - B||

### CODE REVIEW

**Comment by jph00** [comment]
Does it work with this change?

**Comment by bearpelican** [comment]
I haven't tried. I believe @tmabraham will test it out

**Comment by tmabraham** [comment]
@bearpelican Sorry I have been a little busy but I will definitely test it out early next week.
